src/App/Fossa/FossaAPIV1.hs

```diff
@@ -323,8 +329,9 @@ uploadContributors :: (Has (Lift IO) sig m, Has Diagnostics sig m) => ApiOpts ->
 uploadContributors apiOpts locator contributors = fossaReq $ do
   (baseUrl, baseOpts) <- useApiOpts apiOpts
 
-  let opts = baseOpts
-        <> "locator" =: locator
+  let opts =
+        baseOpts
+          <> "locator" =: locator
```

Comment:
[nit]: I think the formatter should have one-lined this.  Let's trick it into doing that:
```suggestion
  let opts = baseOpts <> ("locator" =: locator)
```

Reply:
I agree, but ormolu did this everywhere it could. If there is a way we can force this that would be much better.

Reply:
Ormolu tries to respect whitespace and newlining, so if we don't use newlines, it won't force us unless it thinks it has to.  The original was two lines, so ormolu went full-throttle on newlines.  But if we shorten it to one line, and then format again, I think it will let it go through.